### PR TITLE
Path: typo fix

### PR DESCRIPTION
**Pull request by luzpaz**
[skip ci]

### CODE REVIEW

File: src/Mod/Path/App/AreaPyImp.cpp [hunk 1]
```diff
@@ -160,7 +160,7 @@ static const PyMethodDef areaOverrides[] = {
     {
         "abort",(PyCFunction)areaAbort, METH_VARARGS|METH_KEYWORDS|METH_STATIC,
         "abort(aborting=True): Static method to abort any ongoing operation\n"
-        "\nTo ensure no stray abortion is left in the previous operaion, it is advised to manually clear\n"
+        "\nTo ensure no stray abortion is left in the previous operation, it is advised to manually clear\n"
```
Comment (luzpaz) [inline src/Mod/Path/App/AreaPyImp.cpp]: `abortion` sounds really _off_ but I looked it up and one of it's several meanings is:  
`arrest of development (as of a part or process) resulting in imperfection; also :  a result of such arrest`  
So technically it is correct. Does anyone have any input ?

Comment (luzpaz) [inline src/Mod/Path/App/AreaPyImp.cpp]: @sliptonic care to weigh in?

Comment (sliptonic) [inline src/Mod/Path/App/AreaPyImp.cpp]: Actual fix is trivial typo change.  No problem.  The string isn't user-facing and is clear enough to me.
Looks fine.

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**